plugins/c8y_log_plugin/src/main.rs

```diff
@@ -154,6 +125,59 @@ async fn run(
     }
 }
 
+pub async fn process_mqtt_messages(
```

Comment:
See the comment above.
```suggestion
pub async fn process_mqtt_message(
```

Reply:
yes, make sense. Fixed it.